docs/info/long-range.md

```diff
@@ -1,15 +1,15 @@
 ---
 template: main.html
-description: ExpressLRS boasts great range at any given packet rate. Here's what others reached with their aircrafts.
+description: ExpressLRS boasts a great range at any given packet rate. Here's what others reached with their aircraft.
 ---
 
 ![FAQ-Banner](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-hardware/master/img/community.png)
 
-One of the most frequently asked questions that gets asked by people who are interested in, but haven't yet tried ELRS is "How far does it go, and at what packet rate?"
+One of the most frequently asked questions that are asked by people who are interested in, but haven't yet tried ELRS is "How far does it go, and at what power level?
```

Comment:
you forgot a closing quotation mark (") here ... other than this, looking all good